src/alfasim_sdk/_internal/types.py

```diff
@@ -603,7 +603,7 @@ class MyModel:
     the current value and unit.
     Check out the `Scalar documentation from Barril`_ for more details about the usage.
 
-    .. code-block:: bash
+    .. code-block:: python
```

Comment:
Take a look I think there are other sections of this page that are not applying the highligh right e.g. for class `Reference`. Maybe this could be splited the snippet for python code as `code-block:: python` and the one to represent a terminal command as the previous `code-block:: bash`, I'm not proficient enough in this markup language, but I think this could be done, althouh I think keeping just as python block code is ok.

Reply:
Okay, I'll keep that way.